final/assets/testnet-env

```diff
@@ -0,0 +1,9 @@
+VLS_PORT=17701
+VLS_SERIAL_PORT=/dev/ttyACM0
+VLS_NETWORK=testnet
+# If your SOCKET signer is remote, have the proxy listen to all interfaces.
+# Alternatively, set up a secure tunnel to the signer.
+# VLS_BIND=0.0.0.0
+BITCOIND_RPC_URL=http://rpcuser:VLSsigner1@localhost:18332
+GREENLIGHT_VERSION=v0.11.0.1-62-g92cc76a
```

Comment:
best not to hard-code this.  it can be retrieved via `lightningd --version`

Reply:
this file is directly loaded in container environment. to provide a dynamic environment variable we will have to do it at runtime

Reply:
it doesn't have to be dynamic - you can obtain it in a dockerfile RUN step at build time